### PR TITLE
[3494][ADD] purchase_report_payment_term

### PR DESCRIPTION
**Pull request by rinaldifirdaus**
[3494](https://www.quartile.co/web#id=3494&cids=3&menu_id=505&action=1457&model=project.task&view_type=form)

### CODE REVIEW

File: purchase_order_payment_term/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Purchase Order Payment Term",
+    "version": "16.0.1.0.0",
+    "category": "Purchase",
+    "author": "Quartile Limited, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/purchase-workflow",
+    "license": "AGPL-3",
+    "depends": ["base", "purchase"],
```
Comment (yostashiro) [inline purchase_order_payment_term/__manifest__.py]:
```suggestion
    "depends": ["purchase"],
```

File: purchase_order_payment_term/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (yostashiro) [inline purchase_order_payment_term/__manifest__.py]:
```suggestion
# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
```
Please do the same for all headers.

File: purchase_order_payment_term/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+{
+    "name": "Purchase Order Payment Term",
+    "version": "16.0.1.0.0",
+    "category": "Purchase",
+    "author": "Quartile Limited, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/purchase-workflow",
```
Comment (yostashiro) [inline purchase_order_payment_term/__manifest__.py]:
```suggestion
    "website": "https://github.com/OCA/purchase-reporting",
```

**Review by yostashiro — CHANGES_REQUESTED** [review]
@rinaldifirdaus As we just talked, please make the adjustment on RFQ print as well.

**Comment by yostashiro** [comment]
@rinaldifirdaus I suggested a different name in https://github.com/qrtl/axls-custom/pull/38#pullrequestreview-1550040028, so please provide a reason in case you'd like to use a different name.

**Review by yostashiro — APPROVED** [review]
LGTM 👍 

Please follow up to create a PR in the OCA repo.

**Review by kanda999 — APPROVED** [review]

**Pull request merged**